libraries/Microsoft.Bot.Builder/SetSpeakMiddleware.cs

```diff
@@ -86,7 +86,7 @@ private bool HasTag(string tagName, string speakText)
             {
                 var speakSsmlDoc = XDocument.Parse(speakText);
 
-                if (speakSsmlDoc.Root != null && speakSsmlDoc.Root.AncestorsAndSelf().Any(x => x.Name.LocalName.ToLowerInvariant() == tagName))
+                if (speakSsmlDoc != null && speakSsmlDoc.Root.AncestorsAndSelf().Any(x => x.Name.LocalName.ToLowerInvariant() == tagName))
```

Comment:
Good point; if we have multiple `<voice` tags, we have to handle it and wrap it around the `<speak` tag.

Otherwise, from my understanding, I think [Activity.Speak property](https://docs.microsoft.com/en-us/dotnet/api/microsoft.bot.schema.activity.speak?view=botbuilder-dotnet-stable) will always have the `<speak` root node.

Reply:
added null-conditional operator